recording.h

```diff
@@ -2,6 +2,7 @@
 #define RECORDING_H
 
 #include "xdfwriter.h"
+#include "logger.h"
```

Comment:
Could you move the include to recording.cpp?

Reply:
Yes, makes sense.